code/modules/urist/items/uristweapons.dm

```diff
@@ -115,4 +115,95 @@ Please keep it tidy, by which I mean put comments describing the item before the
 	edge = 0
 	lock_picking_level = 2
 
-///obj/item/weapon/gun/launcher/crossbow/bow //gonna mess with this
\ No newline at end of file
+///obj/item/weapon/gun/launcher/crossbow/bow //gonna mess with this
+
+
+//RS Weapons
+
+
+
+/obj/item/weapon/BronzeDagger
+	name = "Bronze Dagger"
+	desc = "Short but pointy."
+	icon = 'icons/urist/items/uristweapons.dmi'
+	icon_state = "bronze dagger"
+	item_state = "knife"
+	force = 12
+	throwforce = 5
+	hitsound = 'sound/weapons/bladeslice.ogg'
```

Comment:
also, to cut down on repeated code, you could just path them as /obj/item/weapon/urist/blade/etc. and set the parent (/obj/item/weapon/urist/blade) to have the hitsounds and attack verbs and sharp + edge in order to save on repeating code. Not super necessary, but a good piece of advice for the future. Children inherit all the vars and procs of their parents, so anything defined on /obj/item/weapon/urist will be inherited to /obj/item/weapon/urist/bronzesword unless otherwise specified.